packages/flutter/lib/src/animation/curves.dart

```diff
@@ -522,16 +522,136 @@ class Curves {
   /// {@animation 464 192 https://flutter.github.io/assets-for-api-docs/assets/animation/curve_ease_in.mp4}
   static const Cubic easeIn = Cubic(0.42, 0.0, 1.0, 1.0);
 
+  /// A cubic animation curve that starts slowly and ends quickly.
+  ///
+  /// Derived from Robert Penner’s easing functions.
```

Comment:
It would be helpful for each Sine/Quad/Cubic/Quart/Expo/Circ/Back function to have a line explaining why it's named this way.  e.g. "This curve is like [Curves.easeIn], but with sinusoidal easing for a gentler curve".  Penner's chapter on this topic (http://robertpenner.com/easing/penner_chapter7_tweening.pdf) may be helpful here!